### ISSUE TITLE
mwc-dialog does not trigger primaryAction on Enter

### ISSUE THREAD

**Issue by mbinette** [material-components/material-web#1051]
According to the documenation, the `primaryAction` slot element should "Automatically clicked when Enter key is pressed in the dialog." but that does not happen.

**To Reproduce**
1. Create a mwc-dialog with an mwc-button in the primaryAction slot as is defined in the documentation.
2. Open the dialog and press Enter.
3. See that it won't close the dialog or trigger the mwc-button's clicked event.

**Expected behavior**
The Enter button should close the dialog and trigger the primaryAction.

**Browser Version (please complete the following information):**
 - OS: Windows 10
 - Browser chrome, firefox, probably others
 - mwc-dialog Version 0.14.1
 - lit-html Version 1.2.1
 - lit-element Version 2.3.1

**Additional context.
Reproducible on the current demo page as well.

**Comment by 43081j** [comment]
This is because the dialog root is a div and can't receive focus. It won't fire keydown events.

The problem also exists in the MDC repo's [alert demo](https://material-components.github.io/material-components-web-catalog/#/component/dialog).

Can either:
* Give the root div a tabindex
* Do the 'enter triggers primary action' login in the document keydown handler, rather than the root's handler (i'd do this one)

**Comment by asyncLiz** [comment]
Should be fixed in M3
